lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -813,6 +823,22 @@ def test_question_content(self):
         non_endorsed_actual = self.get_comment_list(thread, endorsed=False)
         self.assertEqual(non_endorsed_actual["results"][0]["id"], "non_endorsed_comment")
 
+    def test_endorsed_by_anonymity(self):
```

Comment:
That's covered in test_serializers; I didn't think it was necessary to repeat that here.